functional-tests/devscripts/run_tests.sh

```diff
@@ -113,8 +113,29 @@ if [[ "$PR_CHECK_BUILD" == "true" ]]; then
 	           -e "OPENSHIFT_URL=$OC_CLUSTER_URL" \
 	           -e "OPENSHIFT_USERNAME=$RH_CHE_AUTOMATION_CHE_PREVIEW_USERNAME"  \
 	           -e "OPENSHIFT_PASSWORD=$RH_CHE_AUTOMATION_CHE_PREVIEW_PASSWORD" \
-           quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
-    RESULT=$?
+	       quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
+	RESULT=$?
+	    
+	mkdir screens
```

Comment:
Wouldn't `report` be more in-line with what is used, when running tests locally?

Reply:
Hm, good point, report should be better then screens. Will fix that.